From 0230f06459e41e9c0460de07c2db4aaee9918ba2 Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Thu, 11 Feb 2021 10:22:46 -0600
Subject: [PATCH 22/50] Change name of function that generates event point
 mapping.

This update changes the name of the function that is responsible
for generating the point in the event field be consistent with
the event nomenclature that is now being used.

The debug statement that summarizes the mapping now refers to the
event measurement value as a point.
---
 security/integrity/ima/ima_events.c | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/security/integrity/ima/ima_events.c b/security/integrity/ima/ima_events.c
index ef467c836c7d..3d3581265a35 100644
--- a/security/integrity/ima/ima_events.c
+++ b/security/integrity/ima/ima_events.c
@@ -1533,11 +1533,11 @@ int ima_get_subject(struct ima_event_data *event_data,
 	return retn;
 }
 
-static int ima_map_identity(int function, struct integrity_iint_cache *iint,
-			    struct ima_template_desc *template_desc,
-			    struct file *file, const char *pathname,
-			    struct event_actor *actor,
-			    struct event_subject *subject, char *mapping)
+static int ima_map_event(int function, struct integrity_iint_cache *iint,
+			 struct ima_template_desc *template_desc,
+			 struct file *file, const char *pathname,
+			 struct event_actor *actor,
+			 struct event_subject *subject, char *mapping)
 {
 	int retn, num_fields;
 	struct ima_template_entry *entry;
@@ -1613,8 +1613,8 @@ int ima_events_is_mapped(int function, struct integrity_iint_cache *iint,
 	if (have_violation(file, iint, pathname))
 		return 1;
 
-	if (ima_map_identity(function, iint, template_desc, file, pathname,
-			     &actor, &subject, mapping) != 0)
+	if (ima_map_event(function, iint, template_desc, file, pathname,
+			  &actor, &subject, mapping) != 0)
 		return 0;
 
 	ima_set_actor_status(function, pathname, &subject, mapping);
@@ -1652,7 +1652,7 @@ int ima_events_is_mapped(int function, struct integrity_iint_cache *iint,
 
 
 	if (!retn)
-		pr_debug("[%s]: Mapped entry: comm=%s, pid=%d, func=%d, pathname=%s, fsmagic=%lx, inum=%lu, identity=%*phN\n",
+		pr_debug("[%s]: Mapped event: comm=%s, pid=%d, func=%d, pathname=%s, fsmagic=%lx, inum=%lu, point=%*phN\n",
 			__func__, current->comm, current->pid, function,
 			 pathname, iint->inode->i_sb->s_magic,
 			 iint->inode->i_ino, WP256_DIGEST_SIZE, mapping);
-- 
2.31.1

